AmplifyPlugins/Internal/Sources/InternalAWSPinpoint/Support/Utils/PinpointRequestsRegistry.swift

```diff
@@ -29,11 +29,11 @@ import AWSPluginsCore
         )
     }
 
-    fileprivate func sources(for api: API) -> Set<AWSPinpointSource> {
+    func sources(for api: API) -> Set<AWSPinpointSource> {
```

Comment:
is this change intended?

Reply:
yes, it's being used/accessed by `CustomPinpointHttpClientEngine` which is defined in the same file.  Swiftlint recommended changing it from `fileprivate` to `private` but that would break `CustomPinpointHttpClientEngine`.